Docs (#67)
yiisoft/hydrator: docs/guide/en/creating-own-attributes.md

        private int $min = 0,
        private int $max = 99,
    ) {
    }

    public function getMin(): int
    {
        return $this->min;
    }

    public function getMax(): int
    {
        return $this->max;
    }

    public function getResolver(): string
    {
        return RandomIntResolver::class;
    }
}
```

Resolver:

```php
use Yiisoft\Hydrator\Attribute\Parameter\ParameterAttributeInterface;
use Yiisoft\Hydrator\Attribute\Parameter\ParameterAttributeResolverInterface;
use Yiisoft\Hydrator\AttributeHandling\Exception\UnexpectedAttributeException;
use Yiisoft\Hydrator\AttributeHandling\ParameterAttributeResolveContext;
use Yiisoft\Hydrator\Result;

final class RandomIntResolver implements ParameterAttributeResolverInterface
{
    public function getParameterValue(
        ParameterAttributeInterface $attribute,
        ParameterAttributeResolveContext $context
    ): Result {
        if (!$attribute instanceof RandomInt) {
            throw new UnexpectedAttributeException(RandomInt::class, $attribute);
        }

        $value = rand($attribute->getMin(), $attribute->getMax());

        return Result::success($value);
    }
}
```

> ## Usage one class for attribute and resolver

Reviewer: ```suggestion
## Using a single class for both attribute and resolver
```

> If your attribute simple and don't required dependencies, you can combine attribute and his resolver in one class.

Reviewer: ```suggestion
If your attribute is simple and doesn't require dependencies, you can combine attribute and its resolver in a single class.
```
